src/platform/i386/boot_comp.c

```diff
@@ -112,7 +115,7 @@ kern_boot_comp(void)
         struct captbl *ct;
         unsigned int i;
 	u8_t *boot_comp_captbl;
-	pgtbl_t pgtbl = (pgtbl_t)chal_va2pa(&boot_comp_pgd);
+	pgtbl_t pgtbl = (pgtbl_t)chal_va2pa(&boot_comp_pgd), local_pgd;
```

Comment:
No, this is used for the new page table we created to track local system memory.
See code L161 in the same file.
But if we decide to only use the "SELF_PT", I can remove it.